installer/parse_install_yaml.py

```diff
@@ -164,6 +163,7 @@ def get_distro_item(item: Mapping, key: str, release_version: str, release_type:
                 return {"system_packages": system_packages, "commands": commands}
 
             elif install_type == "ros":
+                ros_release = environ["TUE_ROS_DISTRO"]
```

Comment:
Now this is done multiple times, instead of once.

Reply:
Having it global has a different implication. If TUE_ROS_DISTRO is not set, then our whole tool breaks. This variable is only required while installing a ros system package

Reply:
But I think we require TUE_ROS_DISTRO to be set anyway in multiple spots.

Reply:
We don't. Until ros packages are installed, it is not required

Reply:
But we do always set it in the `boostrap.bash`

Reply:
I know and that becomes a problem when running multiple distros on the same machine. My fix to this to have only tue-env setup on the base computer, create different tue-env workspaces which have docker compose scripts specific to that workspace and then run docker containers from them where TUE_ROS_DISTRO is setup through the bootstrap script.